bundles/org.openhab.core.automation.module.script.rulesupport/src/main/java/org/openhab/core/automation/module/script/rulesupport/loader/AbstractScriptFileWatcher.java

```diff
@@ -83,7 +83,7 @@ public abstract class AbstractScriptFileWatcher implements WatchService.WatchEve
     private final ScriptEngineManager manager;
     private final ReadyService readyService;
     private final WatchService watchService;
-    private final Path watchPath;
+    protected final Path watchPath;
```

Comment:
If the propery never needs to be written adding a getter might be better to prevent future issues.

Reply:
Done